topostats/default_config.yaml

```diff
@@ -33,17 +33,10 @@ grains:
   # Thresholding by height
   grain_crop_padding: 1 # Padding to apply to grains. Needs to be at least 1, more padding may help with unets.
   threshold_method: std_dev # Options : std_dev, otsu, absolute, unet
-  otsu_threshold_multiplier: 1.0
-  threshold_std_dev:
-    below: [10.0] # Thresholds for grains below the image background. List[float].
-    above: [1.0] # Thresholds for grains above the image background. List[float].
-  threshold_absolute:
-    below: [-1.0] # Thresholds for grains below the image background. List[float].
-    above: [1.0] # Thresholds for grains above the image background. List[float].
-  direction: above # Options: above, below, both (defines whether to look for grains above or below thresholds or both)
-  area_thresholds:
-    above: [300, 3000] # above surface [Low, High] in nm^2 (also takes null)
-    below: [null, null] # below surface [Low, High] in nm^2 (also takes null)
+  otsu_threshold_multiplier: 1.0 # Thresholds for data
+  threshold_std_dev: [1.0] # Thresholds for data either side of the image background, positive for above and negative for below
+  threshold_absolute: [1.0] # Thresholds for data either side of the image background, positive for above and negative for below
```

Comment:
As above, could just be a float rather than a list.

This impacts the `Grains` class and the type-hints used there.